lib/shoryuken/environment_loader.rb

```diff
@@ -53,7 +53,9 @@ def config_file_options
     # https://github.com/phstc/shoryuken/blob/a81637d577b36c5cf245882733ea91a335b6602f/lib/shoryuken.rb#L82
     # Please delete this method afert next release (v2.0.12 or later)
     def initialize_aws
-      Shoryuken.logger.warn { "[DEPRECATION] aws in shoryuken.yml is deprecated. Please use configure_server and configure_client in your initializer"} unless Shoryuken.options[:aws].nil?
+      unless Shoryuken.options[:aws].to_h.empty?
+        Shoryuken.logger.warn { '[DEPRECATION] aws in shoryuken.yml is deprecated. Please use configure_server and configure_client in your initializer' }
+      end
```

Comment:
@h3poteto @suzan2go I have a project which I don't use Rails, I use a standalone version of Shoryuken. So for this project, I would need to create a Ruby class, and make sure it's the first file required in the `-r` to set the AWS config. Also, there are still some people that during the deploy process (capistrano), copy a `shoryuken.yml` to the project's dir, for this people they would need to copy a class instead, which feels bad or change the way they initialize AWS. 

> Because if the configuration of aws is in shoryuken.yml, the initialize process of shoryuken becomes too complicated.

Unless I'm missing something I think the only thing it requires is this:
https://github.com/phstc/shoryuken/blob/master/lib/shoryuken/environment_loader.rb#L55-L58

WDYT?